rolling-shutter/mocksequencer/shutter.go

```diff
@@ -25,7 +27,43 @@ func (s *ShutterService) GetBatchIndex() (string, error) {
 	return hexutil.EncodeUint64(s.processor.batchIndex), nil
 }
 
-func (s *ShutterService) SubmitBatch(transaction string) (string, error) {
+func (s *ShutterService) validateBatch(tx *txtypes.Transaction) error {
+	if tx.Type() != txtypes.BatchTxType {
+		return errors.New("unexpected transaction type")
+	}
+	// TODO retrieve the collator for that block number (which could be higher then the last set
+	// "activation blocknumber")
+	if tx.Gas() > GasLimit {
```

Comment:
Batch txs don't have a gas field (we might want to check the total gas of all transactions though)